src/app/portlets/dot-edit-page/dot-edit-page-routing.module.ts

```diff
@@ -5,30 +5,31 @@ import { DotEditPageMainComponent } from './main/dot-edit-page-main/dot-edit-pag
 // tslint:disable-next-line:max-line-length
 import { DotLegacyTemplateAdditionalActionsComponent } from './layout/components/dot-template-additional-actions/dot-legacy-template-additional-actions-iframe/dot-legacy-template-additional-actions-iframe.component';
 import { EditLayoutResolver } from './layout/services/dot-edit-layout-resolver/dot-edit-layout-resolver.service';
-import { EditContentResolver } from '../dot-edit-content/services/dot-edit-content-resolver.service';
+import { EditContentResolver } from './content/services/dot-edit-content-resolver.service';
 
 const dotEditPage: Routes = [
     {
         component: DotEditPageMainComponent,
         path: '',
         resolve: {
-            pageView: EditLayoutResolver
+            content: EditContentResolver
```

Comment:
Because normally we'll going to edit the page before the layout is better to do it this way.